main.rb

```diff
@@ -0,0 +1,44 @@
+require './app'
+
+def options
+  print "\nPlease choose an option by entering a number \n\n"
+  print "1 - List all books \n"
+  print "2 - List all people \n"
+  print "3 - Create a person (teacher or student) \n"
+  print "4 - Create a book \n"
+  print "5 - Create a rental \n"
+  print "6 - List all rentals for a person \n"
+  print "7 - Quit \n"
```

Comment:
- [ ] The app closes if the user clicks `enter` two times, kindly fix this so the user can exit the app only when selecting option `7`

Reply:
main.rb:15:1: C: Metrics/CyclomaticComplexity: Cyclomatic complexity for functional is too high. [8/7]
def functional(app)

after i rectified the issue you raised, the linters flagged the input as too high